src/statement.rs

```diff
@@ -482,13 +508,19 @@ impl<'a> StatementRef<'a> {
         self.statement.group_end != NO_STATEMENT
     }
 
+    /// Obtain the span corresponding to the statment label.
+    #[must_use]
+    pub const fn label_span(&self) -> Span {
+        self.statement.label
+    }
```

Comment:
The compiler will almost certainly inline it, but it doesn't hurt to put `#[inline]` anyway. Although I'm told that there is actually a difference when using `#[inline]` on public library methods, because non-inline methods from an external crate can't be inlined because their body is not available to the compiler, so you will often see copious `#[inline]` in libraries like std for this reason.

Reply:
Excellent, thanks!